content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -108,32 +110,40 @@ cluster for mesh expansion, run the following commands on a machine with cluster
           -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
```

Comment:
We should add some introductory text about SERVICE_NAMESPACE. It's used here for the first time with no mention about what it means.

Reply:
This is mentioned in step 1 now.